facilitator/src/bin/facilitator.rs

```diff
@@ -1032,29 +1076,20 @@ fn aggregate_subcommand(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
         sub_matches.value_of("aggregation-end").unwrap(),
         batch_info,
         sub_matches,
+        None,
         || {}, // no-op callback
     )
 }
 
 fn aggregate_worker(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
-    let aggregation_started: Counter = register_counter!(
-        "aggregation_jobs_begun",
-        "Number of aggregation jobs that started (on the facilitator side)"
-    )
-    .context("failed to register counter for started aggregations")?;
-    let aggregation_finished = register_counter_vec!(
-        "aggregation_jobs_finished",
-        "Number of aggregation jobs that finished (on the facilitator side)",
-        &["status"]
-    )
-    .context("failed to register counter for finished aggregations")?;
-
     let mut queue = aggregation_task_queue_from_args(sub_matches)?;
+    let metrics_collector = AggregateMetricsCollector::new()?;
+    let scrape_port = value_t!(sub_matches.value_of("metrics-scrape-port"), u16)?;
+    let _runtime = start_metrics_scrape_endpoint(scrape_port)?;
```

Comment:
```suggestion
    start_metrics_scrape_endpoint(scrape_port)?;
```

Reply:
You have to keep a live reference to the runtime or it gets dropped, deallocated, and then tasks spawned onto it never run. Admittedly I should have a comment explaining that surprising behaviour. I'd also be very happy to learn that I'm just wrong.

Reply:
Oh right, that makes sense with the comment on the function. You could potentially use [`std::mem::forget()`](https://doc.rust-lang.org/std/mem/fn.forget.html) to make this explicit. But I guess the current behavior actually does run the runtime's Drop when the enclosing functions exits, which is probably nicer and tidier.